spanner/spannertest/db_eval.go

```diff
@@ -449,11 +451,36 @@ func (ec evalContext) evalExpr(e spansql.Expr) (interface{}, error) {
 	}
 }
 
-func (ec evalContext) evalID(id spansql.ID) (interface{}, error) {
-	for i, col := range ec.cols {
-		if col.Name == id {
-			return ec.row.copyDataElem(i), nil
+// resolveColumn turns an ID or PathExp into a table column.
+func (ec evalContext) resolveColumn(e spansql.Expr) (int, error) {
```

Comment:
nit: wouldn't `resolveColumnIndex` be a better function name now?

Reply:
Yeah, fair enough. Done.